charts/pulsar/values.yaml

```
@@ -383,6 +383,10 @@ zookeeper:
         # type: pd-ssd
         # fsType: xfs
         # provisioner: kubernetes.io/gce-pd
+      ## If you want to bind static persistent volumes via selectors, e.g.:
+      # selectors:
+        # matchLabels:
+        # app: pulsar-zookeeper
```

I think it is slightly more helpful to have a good zero value here. Also, we need `selector` not `selectors`.

```suggestion
      ## If you want to bind static persistent volumes via selectors, e.g.:
      selector: {}
```

---

I've made the changes, please review